coderd/database/migrations/000065_app_slug.up.sql

```diff
@@ -7,3 +9,8 @@ UPDATE "workspace_apps" SET "slug" = "name";
 -- make "slug" column not nullable and remove default
 ALTER TABLE "workspace_apps" ALTER COLUMN "slug" SET NOT NULL;
 ALTER TABLE "workspace_apps" ALTER COLUMN "slug" DROP DEFAULT;
+
+-- add unique index on "slug" column
+ALTER TABLE "workspace_apps" ADD CONSTRAINT "workspace_apps_agent_id_slug_key" UNIQUE ("agent_id", "slug");
```

Comment:
```suggestion
ALTER TABLE "workspace_apps" ADD CONSTRAINT "workspace_apps_agent_id_slug_idx" UNIQUE ("agent_id", "slug");
```

Reply:
wait nevermind I'm thinking of the wrong file

Reply:
fixed